### DISCUSSION TITLE
How to develop with Android-Debug-Database

### DISCUSSION THREAD

**Discussion by greenking19** [simolus3/drift#1889] [Q&A] [answered]
I found this tool on github, but it doesn't seem like I can use it with drift
👉 [Android-Debug-Database](https://github.com/amitshekhariitbhu/Android-Debug-Database)

**Comment by simolus3** [comment, marked as answer]
Interesting tool! Yeah, it's not possible to use that with a `NativeDatabase` because the implementation sidesteps Android's database APIs. You _could_ use a `SqfliteDatabase` from `package:drift_sqflite` in debug builds if you want to use that tool.

Alternatively, you can use the [storage_inspector](https://pub.dev/packages/storage_inspector) or [drift_db_viewer](https://pub.dev/packages/drift_db_viewer) packages, which are similar tools that work great with drift.

**Reply by greenking19** [reply]
Thank you! 
I just found out that there is the package drift_db_viewer. 
Thank you very much, it meets my needs!
